app/models/mixins/event_mixin.rb

```diff
@@ -36,6 +43,15 @@ def events_table_name(assoc)
     events_assoc_class(assoc).table_name
   end
 
+  def event_where_clause(assoc = :ems_events)
+    case assoc.to_sym
+    when :ems_events, :event_streams
+      event_where_clause_ems_events
+    when :miq_events, :policy_events
```

Comment:
I don't know if the existing classes wanted policy_events to return PolicyEvent rows (policy success/failure) or MiqEvent records.  🤷

Reply:
I still have to fix this.